certbot/main.py

```diff
@@ -779,11 +779,45 @@ def install(config, plugins):
     except errors.PluginSelectionError as e:
         return str(e)
 
-    domains, _ = _find_domains_or_certname(config, installer)
-    le_client = _init_le_client(config, authenticator=None, installer=installer)
-    _install_cert(config, le_client, domains)
-
-
+    # If cert-path is defined, populate missing (ie. not overridden) values.
+    # Unfortunately this can't be done in argument parser, as certificate
+    # manager needs the access to renewal directory paths
+    if config.certname:
+        config = _populate_from_certname(config)
+    if config.key_path and config.cert_path:
+        _check_certificate_and_key(config)
+        domains, _ = _find_domains_or_certname(config, installer)
+        le_client = _init_le_client(config, authenticator=None, installer=installer)
+        _install_cert(config, le_client, domains)
+    else:
+        raise errors.ConfigurationError("Path to certificate or key was not defined. "
+            "If your certificate is managed by Certbot, please use --cert-name "
+            "to define which certificate you would like to install. "
+            "Alternatively you can use both --key-path and --cert-path to install a "
```

Comment:
Normally people would want to use `--fullchain-path` in addition to `--cert-path`, right?

Reply:
(See also #1930)

Reply:
It might even be best to not document that alternative way of using `--install`, because it's so weird, confusing and broken?

Reply:
In summary, either add `--fullchain-path` to this CLI documentation, or remove the sentence starting "Alternatively..."?

Reply:
I decided to remove the later part of the message to keep the message short and concise. And there certainly is broken parts in the later execution regarding manually defining all the different values.